include/bout/region.hxx

```diff
@@ -90,18 +90,21 @@
     for (auto index = block->first; index < block->second; ++index)
 
 #ifdef _OPENMP
-#define BLOCK_REGION_LOOP_SECTION(index, region, omp_pragmas)                            \
+#define BLOCK_REGION_LOOP_OMP(index, region, omp_pragmas)                                \
   BOUT_OMP(omp_pragmas)                                                                  \
   for (auto block = region.getBlocks().cbegin(); block < region.getBlocks().cend();      \
        ++block)                                                                          \
     for (auto index = block->first; index < block->second; ++index)
 #else
 // No OpenMP, so fall back to slightly more efficient serial form
-#define BLOCK_REGION_LOOP_SECTION(index, region, omp_pragmas)                            \
+#define BLOCK_REGION_LOOP_OMP(index, region, omp_pragmas)                                \
   BLOCK_REGION_LOOP_SERIAL(index, region)
 #endif
 
-#define BLOCK_REGION_LOOP(index, region) BLOCK_REGION_LOOP_SECTION(index, region, parallel for)
+#define BLOCK_REGION_LOOP(index, region) BLOCK_REGION_LOOP_OMP(index, region, parallel for)
```

Comment:
I think this should have `schedule (guided)` as well.

Reply:
Thanks, fixed